java/ql/src/Security/CWE/CWE-352/CsrfUnprotectedRequestType.qhelp

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE qhelp SYSTEM "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>When you set up a web server to receive a request from a client without any mechanism
+for verifying that it was intentionally sent, then it is vulnerable to a Cross Site Request
```

Comment:
Not sure whether we should use a hyphen here but you've been using one in the change note and in the description in the ql file, so adding one here for consistency.

```suggestion
for verifying that it was intentionally sent, then it is vulnerable to a Cross-Site Request
```